backend/src/main/java/com/handong/rebon/review/domain/content/ReviewContent.java

```diff
@@ -1,10 +1,50 @@
 package com.handong.rebon.review.domain.content;
 
+import com.handong.rebon.exception.review.ReviewContentFormatException;
+import com.handong.rebon.exception.review.ReviewTitleFormatException;
+import lombok.Getter;
+
 import javax.persistence.Embeddable;
+import java.util.Objects;
 
+@Getter
 @Embeddable
 public class ReviewContent {
+
     private String title;
     private String content;
     private String tip;
+
+    protected ReviewContent() {
+    }
+
+    public ReviewContent(String title, String content) {
+        IsNotValidtitleAndContent(title, content);
+        this.title = title;
+        this.content = content;
+    }
+
+    public ReviewContent(String title, String content, String tip) {
+        this(title,content);
+        this.tip = tip;
+    }
```

Comment:
지금은 각 생성자에서 할당 작업이 일어나고 있는 것 같은데요. 생성자를 여러개 둘 것이면 초기화 로직은 하나의 주생성자에서만 처리하고 부 생성자들이 주생성자를 호출하는 방법은 어떤가요?

```suggestion
    public ReviewContent(String title, String content) {
        this(title, content, "");
    }

    public ReviewContent(String title, String content, String tip) {
        IsNotValidtitleAndContent(title, content);
        this.title = title;
        this.content = content;
        this.tip = tip;
    }
```

Reply:
저는 생성자 위에 @Builder 를 추가하여 public ReviewContent(String title, String content, String tip) 이거 하나로 생성자를 줄일 수 있었습니다.